internal/clicontext/config.go

```diff
@@ -62,6 +62,12 @@ func (c *Config) FromURL(v string) error {
 	// Override
 	if u.Host != "" {
 		c.Server.Address = u.Host
+
+		// If no port is specified, default to port 9701 which is our default
+		// gRPC port for Waypoint installations.
+		if idx := strings.IndexByte(u.Host, ':'); idx < 0 {
+			c.Server.Address += ":9701"
```

Comment:
Should we do something like....

```suggestion
			c.Server.Address += fmt.Sprintf(":%s", serverinstall.defaultGrpcPort)
```

Maybe the source of truth should move outside of serverinstall but it would be nice if we had a place that defined these defaults for things like this to use in case we change it in the future 🤔

Reply:
Note I can't accept your suggestion because its not exported. We can export it and perhaps use that though.

Reply:
Ah yeah, I should of done a code block 😄 I figured it might break too since the suggestion doesn't import `severinstall` too

Reply:
Actually no, this creates an import cycle so this can't be the source of truth. We use this port ALL OVER so I'd say we just keep it hardcoded and figure it out later.

Reply:
What if we place it in `internal/serverconfig/config`? https://github.com/hashicorp/waypoint/blob/main/internal/serverconfig/config.go

Reply:
Done!